translate/filters/checks.py

```diff
@@ -2371,6 +2376,17 @@ def run_filters(self, unit, categorised=False):
 
         return result
 
+    @critical
+    def l20n_validation(self, str1, str2):
+        """Checks for valid l20n format."""
```

Comment:
Pootle extracts the check description from this docstring, so please add a verbose description that tells translators what this check actually does, like https://mozilla.locamotion.org/help/quality-checks/#xmltags